src/services/googleDriveManager.js

```
@@ -0,0 +1,660 @@
+            "Parent folder not found. Please select a new folder.",
+          );
+        } else {
+          // その他の作成エラー
+          console.error("Error creating new file:", error);
+          if (error.result && error.result.error) {
+            console.error("Detailed error:", error.result.error.message);
+          }
+          return null;
+        }
+      }
+    }
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)